src/app/app-constants.ts

```diff
@@ -0,0 +1,3 @@
+export class AppConstants {
```

Comment:
I don't think you should create a class for this constant, it can just be a standalone constant

Reply:
Done